### PR TITLE
Conclave, News and Notification

### LINKED ISSUE #434: Notification Settings for Requiem Mods
Summary (short):
Add Notification Settings for Requiem Mods

https://imgur.com/a/PkjNwwt

### PR DESCRIPTION
**Pull request by EricSihaoLin**
**Summary:**

Several fixes that I've been wanting to implement

- Trimming of notifiables on fissures and inclusion of Requiem Fissures
- Conclave Panel (default not displayed)
- News Panel fixes for live events and relative time

---
**Fixes issue (include link):**

Fixes #434 

---
**Mockups, screenshots, evidence:**

News Panel (no more string splicing, now using moment.js):
image

Conclave Panel:
image

Notifiables:
image

### CODE REVIEW

File: src/components/panels/SentientOutpostsPanel.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default {
   name: 'SentientOutpostsPanel',
   props: ['sentientOutposts'],
   watch: {
-    mission: function () {
+    mission: function() {
```
Comment (EricSihaoLin) [inline src/components/panels/SentientOutpostsPanel.vue:42]: Codacy Issue found: [Insert `·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5567676)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5567676&bid=18259173)

File: src/components/panels/ConclavePanel.vue [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    activeChallenges() {
+      return this.conclave
+        .filter((challenge) => !challenge.rootChallenge)
+        .sort((a, b) => {
+          return a.category == 'week' || b.category == 'week' ? 1 : a.mode.localeCompare(b.mode);
+        });
+    },
+  },
+  methods: {
+    type: function(challenge) {
```
Comment (EricSihaoLin) [inline src/components/panels/ConclavePanel.vue:64]: Codacy Issue found: [Insert `·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5567676)

File: src/components/panels/ConclavePanel.vue [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        return this.$t('conclave.cephaloncapture');
+      } else if (challenge.mode == 'Lunaro') {
+        return this.$t('conclave.lunaro');
+      } else if (challenge.category == 'week') {
+        return this.$t('conclave.weekly');
+      } else {
+        return this.$t('conclave.header');
+      }
+    },
+    image: function(challenge) {
```
Comment (EricSihaoLin) [inline src/components/panels/ConclavePanel.vue:79]: Codacy Issue found: [Insert `·`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5567676)

File: src/main.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ import './registerServiceWorker';
 Vue.config.productionTip = false;
 
 /* Sentry Reporting */
+/*
```
Comment (TobiTenno) [inline src/main.js]: uncomment

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**